docs/source/upgrade-guide/index.md

```diff
@@ -253,31 +253,62 @@ It is highly recommended that you use latest Node 16.
 2. Change your root project `Makefile` to include these commands:
 
 ```diff
```

Comment:
This diff needs to be indented so that it is part of the numbered list.

Hint: I always indent my list items to 4 spaces, so that when I need to indent code blocks, I can select the selection and hit tab to indent it properly.

Reply:
All indentations fixed. What do you mean by "I always indent my list items to 4 spaces"? the whole 1. xxxx element?  the markdown linter complains if I do so. or just the text?

Reply:
Like so, where on the first line, `Item 1` is indented 4 spaces, counting the `1`, `.`, and two spaces:

````md
1.  Item 1
1.  Item 2

    a.  Item 2.a.

        ```shell
        rm -rf  # twitter lulz
        ```

1.  Item 3
````

Some code editors get annoying with their auto-correct or auto-formatting when you mix the number of spaces between levels of indentation. The editor sometimes inconsistently forces 2, 3, or 4 spaces on indents, or a mixture of them, especially on new lines or select, then tab to indent.